docs/UserGuide.md

```diff
@@ -8,43 +8,54 @@ title: User Guide
 * [3. Quick start](#3-quick-start)
-* [4. Features](#4-features)
-  * [4.1. Basic Commands](#41-basic-commands)
-    * [4.1.1. Adding a food item](#411-adding-a-food-item)
-    * [4.1.2. Editing a food item](#412-editing-a-food-item)
-    * [4.1.3. Deleting a food item](#413-deleting-a-food-item)
-    * [4.1.4. Changing the quantity of a food item](#414-changing-the-quantity-of-a-food-item)
-  * [4.2. Sorting Commands](#42-sorting-commands)  
-    * [4.2.1. Sort food items by description](#421-sort-food-items-by-description)
-    * [4.2.2. Sort food items by expiry date](#422-sort-food-items-by-expiry-date)
-    * [4.2.3. Sort food items by priority](#423-sort-food-items-by-priority)
-  * [4.3. Viewing Commands](#43-viewing-commands)    
-    * [4.3.1. Listing all food items](#431-listing-all-food-items)
-    * [4.3.2. Finding food items](#432-finding-food-items)
-    * [4.3.3. Viewing expired food items](#433-viewing-expired-food-items)
-  * [4.4. Miscellaneous Commands](#44-miscellaneous-commands)
-    * [4.4.1. Undoing previous command](#441-undoing-previous-command)
-    * [4.4.2. Redoing previously undone command](#442-redoing-previously-undone-command)     
-    * [4.4.3. Clearing all entries](#443-clearing-all-entries)
-    * [4.4.4. Viewing help](#444-viewing-help)
-    * [4.4.5. Exiting the program](#445-exiting-the-program)
-  * [4.5. Saving the data](#45-saving-the-data)
-* [5. FAQs](#5-faq)
-* [6. Command summary](#6-command-summary)
-* [7. Glossary](#7-glossary)
+* [4. Application Layout](#4-application-layout)
+* [5. Features](#5-features)
+  * [5.1. Basic Commands](#51-basic-commands)
+    * [5.1.1. Adding a food item](#511-adding-a-food-item)
+    * [5.1.2. Editing a food item](#512-editing-a-food-item)
+    * [5.1.3. Deleting a food item](#513-deleting-a-food-item)
+    * [5.1.5. Changing the quantity of a food item](#514-changing-the-quantity-of-a-food-item)
+  * [5.2. Sorting Commands](#52-sorting-commands)  
+    * [5.2.1. Sort food items by description](#521-sort-food-items-by-description)
+    * [5.2.2. Sort food items by expiry date](#522-sort-food-items-by-expiry-date)
+    * [5.2.3. Sort food items by priority](#523-sort-food-items-by-priority)
+  * [5.3. Viewing Commands](#53-viewing-commands)    
+    * [5.3.1. Listing all food items](#531-listing-all-food-items)
+    * [5.3.2. Finding food items](#532-finding-food-items)
+    * [5.3.3. Viewing expired food items](#533-viewing-expired-food-items)
+  * [5.4. Miscellaneous Commands](#54-miscellaneous-commands)
+    * [5.4.1. Undoing previous command](#541-undoing-previous-command)
+    * [5.4.2. Redoing previously undone command](#542-redoing-previously-undone-command)     
+    * [5.4.3. Clearing all entries](#543-clearing-all-entries)
+    * [5.4.4. Viewing help](#544-viewing-help)
+    * [5.4.5. Exiting the program](#545-exiting-the-program)
+  * [5.5. Saving the data](#55-saving-the-data)
+* [6. Upcoming Features](#6-upcoming-features)
```

Comment:
Update link to match full header. Currently not does not go to section.

Reply:
Thanks!